cli/auth.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/pretty"
+	"github.com/coder/serpent"
+)
+
+func (r *RootCmd) auth() *serpent.Command {
+	cmd := &serpent.Command{
+		Use:   "auth <subcommand>",
+		Short: "Manage information about internal authentication.",
```

Comment:
Maybe something like:

```suggestion
		Short: "Authenticate with a Coder deployment and view authentication status.",
```

Or:

```suggestion
		Short: "Manage authentication for Coder deployment.",
```

I feel "internal" here could be confusing to users.